libs/components/flyout/src/lib/modules/flyout/flyout.service.ts

```diff
@@ -42,8 +41,7 @@ export class SkyFlyoutService implements OnDestroy {
     private windowRef: SkyAppWindowRef,
     private dynamicComponentService: SkyDynamicComponentService,
     private router: Router,
-    private readonly _ngZone: NgZone,
-    private readonly applicationRef: ApplicationRef
```

Comment:
Removing this argument will cause problems for consumers that mock the flyout service in their tests: https://dev.azure.com/blackbaud/_search?action=contents&type=code&text=provide%3A%20SkyFlyoutService&result=DefaultCollection/Products/skyux-spa-reportview/GBmaster//src/app/components/dashboard-view/dashboard-view.component.spec.ts

Reply:
Added back as `Optional`